cobc/cobc.c

```diff
@@ -6741,6 +6783,146 @@ print_fixed_line (const int line_num, char pch, char *line)
 	}
 }
 
+static int
```

Comment:
Please add a doc comment before the function so it is clear what is checks/changes.

Also: does the code handle an `"escaped "" quote like in x""4100"""`?